src/Service/Terminal/Service_Terminal.rei

```diff
@@ -1,20 +1,18 @@
 open Oni_Core;
 open Oni_Extensions;
 
-module Msg: {
-  type t =
-    // TODO:
-    | Resized(unit)
-    | Updated(unit)
-    | ProcessStarted({
-        id: int,
-        pid: int,
-      })
-    | ProcessTitleSet({
-        id: int,
-        title: string,
-      });
-};
+type msg =
+  // TODO:
+  | Resized(unit)
+  | Updated(unit)
+  | ProcessStarted({
+      id: int,
+      pid: int,
+    })
+  | ProcessTitleSet({
+      id: int,
+      title: string,
+    });
```

Comment:
This is a function call. Why not `ProcessTitleChanged`?

Reply:
Changed to `ProcessTitleChanged`